packages/mobile/src/screens/upload-screen/screens/TrackVisibilityScreen.tsx

```diff
@@ -0,0 +1,56 @@
+import { View } from 'react-native'
+
+import IconHidden from 'app/assets/images/iconHidden.svg'
+import { Divider, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+import { UploadStackScreen } from '../UploadStackScreen'
+import { SwitchField } from '../fields'
+
+const messages = {
+  title: 'Track Visibility',
+  description:
+    "Hidden tracks won't show up on your profile. Anyone who has the link will be able to listen.",
+  hideTrack: 'Hide Track',
+  showGenre: 'Show Genre',
+  showMood: 'Show Mood',
+  showTags: 'Show Tags',
+  showShareButton: 'Show Share Button',
+  showPlayCount: 'Show Play Count'
+}
+
+const useStyles = makeStyles(({ spacing }) => ({
+  content: {
+    padding: spacing(6)
+  },
+  divider: {
+    marginTop: spacing(4),
+    marginHorizontal: spacing(-6)
```

Comment:
(see negative margin comment above)

Reply:
totes, let me know what you think re my comment!